parts/k8s/kubernetescustomscript.sh

```diff
@@ -59,17 +59,23 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
+if [[ $OS != $COREOS_OS_NAME ]]; then
```

Comment:
sorry, more cruft, `upgradeOs` doesn't exist

Reply:
All the same stuff that got me with the rebase the first time 😦